perllib/FixMyStreet/App/Controller/Contact.pm

```diff
@@ -102,6 +102,10 @@ sub determine_contact_type : Private {
         }
     }
 
+    if ( $c->cobrand->abuse_reports_only && not ($id || $token) ) {
```

Comment:
this could be an `elsif ($c->cobrand->abuse_reports_only) {` I think, as then you'd already know there was no id or token.